### PR TITLE
Wrapped into Python package and Added PKGBUILD

### PR DESCRIPTION
**Pull request by zatevakhin**
Changes:

- Wrapped the code into a Python package using `poetry`.
- Added a PKGBUILD file for AUR compatibility.

Testing:
- Run poetry build for Python package.
- Run `makepkg -si` for Arch Linux package.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pkgname=vmvm
+pkgver=0.1.0
+pkgrel=1
+pkgdesc="QEMU wrapper and shell"
+arch=('any')
+url="https://github.com/zatevakhin/vmvm"
+license=('MIT')
```
Comment (roovio) [inline PKGBUILD:7]: GPLv3

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+depends=('python' 'python-yaml')
+makedepends=('git' 'poetry')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/zatevakhin/$pkgname/archive/$pkgver.tar.gz")
```
Comment (rufiorogue) [inline PKGBUILD]: do we need this source?

File: PKGBUILD (hunk 1 above)
Comment (rufiorogue) [inline PKGBUILD]: GPLv3

File: PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
+pkgname=vmvm
```
Comment (rufiorogue) [inline PKGBUILD:1]: vmvm-git?

File: poetry.lock
Comment (rufiorogue) [inline poetry.lock:1]: is this a temporary file?

File: vmvm/lib/tpm_manager.py
Comment (rufiorogue) [inline vmvm/lib/tpm_manager.py:1]: no quote style changes in this PR pls

File: vmvm/vmvm.py [hunk 4]
```diff
@@ -1,14 +1,9 @@
...
 from logging import info,warning,error
-from lib.tpm_manager import TPMManager
-
+from .lib.tpm_manager import TPMManager
 
-def get_local_dir():
-    return os.path.dirname(os.path.realpath(__file__))
 
-presets = yaml.safe_load(open(get_local_dir()+'/presets.yml', 'r'))
+presets = yaml.safe_load(open(os.path.join(os.getcwd(), "presets.yml"), 'r'))
```
Comment (rufiorogue) [inline vmvm/vmvm.py]: Not  a correct path for presets.yml. The file is not a user config, and should be installed to app lib/data dir. Don't confuse with vmconfig.yml which is user editable config to be put in cwd

File: .gitignore
Comment (rufiorogue) [inline .gitignore:1]: only relevant ignores please. Keep it compact and simple.

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+description = "QEMU wrapper and shell"
+authors = ["Roovio"]
+readme = "README.md"
+
+classifiers = [
+    "Intended Audience :: Developers",
+    "Environment :: Console",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
```
Comment (rufiorogue) [inline pyproject.toml]: 3.10

File: PKGBUILD (hunk 3 above)
Comment (zatevakhin) [reply PKGBUILD:1]: There is no need because we are using a specific version from `tag`, not from `master`.

**Pull request merged**